reana_server/rest/analyses.py

```diff
@@ -730,10 +894,11 @@ def get_analysis_outputs_list(analysis_id):  # noqa
               }
     """
     try:
-        response, http_response = rwc_api_client.api.get_workflow_outputs(
+        response, http_response = rwc_api_client.api.get_workflow_files(
             user=request.args.get('user'),
             organization=request.args.get('organization'),
-            workflow_id=analysis_id).result()
+            workflow_id=analysis_id,
+            input_type='output').result()
```

Comment:
The `get_workflow_files` endpoint is used to retrieve files of a workflow / analysis.
The parameter ` input_type` may not be the most descriptive term to specify that one wants to retrieve all files that have been created by a workflow.
Maybe `file_type` or just `type` would be more descriptive or not less confusing compared to `input_type`, but on the other hand `file_type` or `type` is kind of ambiguous. Sorry I don't have better suggestions...